### PR TITLE
Jupyter Branch - This one has mainly focused on getting gtb.py working

### PR DESCRIPTION
**Pull request by jasonbraganza**

### CODE REVIEW

File: gtb.py [hunk 1]
```diff
@@ -1,19 +1,58 @@
...
+        print_date_format = "%a, %Y-%m-%d %H:%M"
+        print_header = f'{"Date":<20s}\t{"Commit Hash":<15s}\t{"Author Name":<25s}{"Branch":<40s}{"Commit Message"}\n'
+
+        # Finally present what we fished from the repo, to the user.
+        print(print_header)
+
+        for key in reversed(sorted(printlog)):
+            print_date = key.strftime(print_date_format)
+            print(
+                f"{print_date:<20s}\t{printlog[key].sha_num[:7]:<15s}\t{str(printlog[key].committer_name):<25s}{printlog[key].branch_name:<40s}{printlog[key].commit_message}"
```
Comment (kushaldas) [inline gtb.py:51]: This works great, but maybe you will have to remove a few columns. 

```
Date                	Commit Hash    	Author Name              Branch                                  Commit Message

Fri, 2020-11-20 18:31	e3aed03        	Kushal Das               fix_dhclient_path_for_testinfra         commit: Updates test strings to verify gpg2 output
Tue, 2020-11-17 17:34	c114575        	Kushal Das               develop                                 pull: Fast-forward
Mon, 2020-11-09 19:34	0b857f5        	Kushal Das               focal_ci                                commit (amend): Fixes #5524 New CI job for app tests on Focal
Wed, 2020-11-04 18:46	993e83a        	Kushal Das               dev_focal                               rebase (finish): refs/heads/dev_focal onto 8a1833d7dfbe9970270e370f9ee8e63858c96f0a
Mon, 2020-11-02 15:27	ba68bfb        	Kushal Das               alembic_migration_fix_due_to_sqlite_upgradecommit: Fixes #5605 alembic migration errors on Focal
Wed, 2020-10-28 17:28	a36821a        	Kushal Das               fix_5603_functional_metatdata_for_focal commit: Fixes #5603 server os can be 16.04 or 20.04
Wed, 2020-10-28 15:16	7cfd897        	Kushal Das               fix_5601_secure_delete_test             commit: Fixes #5601 test_rm failure on Focal
Tue, 2020-10-27 20:20	a11647a        	Kushal Das               updates_pytest_pluggy_testinfra_molecule_and_the_universerebase (finish): refs/heads/updates_pytest_pluggy_testinfra_molecule_and_the_universe onto bb80bfc1171304fb67c1d8ff6a8f3002fd742e66
```

Comment (jasonbraganza) [reply gtb.py:51]: Commit Message? that seems the least important to my mind.

Comment (jasonbraganza) [reply gtb.py:51]: @kushaldas, or should I shorten the date format?

Comment (jasonbraganza) [reply gtb.py:51]: @kushaldas ping

**Pull request closed without merging**